src/main/java/Calculator/CalculatorController.java

```diff
@@ -0,0 +1,21 @@
+package Calculator;
+
+import Calculator.View.*;
+import Calculator.Model.*;
+
+
+public class CalculatorController {
+    public void start() {
+        System.out.println("계산기를 시작합니다.");
+
+        String[] elements = InputView.getAlgebra();
+
+        Calculator calculator = new Calculator();
+        for (String element : elements) {
+            calculator.type(element);
+        }
+
+        System.out.println("계산 결과는 " + calculator.currentResult() + "입니다.");
```

Comment:
```suggestion
        CalculatorRequest request = InputView.input();

        Calculator calculator = new Calculator(request);
        int result = calculator.execute();

        System.out.println("계산 결과는 " + result + "입니다.");
```

위와 같은 틀로 한번 리팩토링해보는건 어떨까요